complaints/settings.json

```diff
@@ -57,6 +57,13 @@
           "lowercase",
           "cr_shingle"
         ]
+      },
+      "uppercase_analyzer": {
+        "type": "custom",
+        "tokenizer": "keyword",
+        "filter": [
+          "uppercase"
```

Comment:
I like the approach of using the analyzer to do the case analysis. I just wanted to confirm though: this will cause Elasticsearch to analyze the field as uppercase, rather than convert the field to uppercase, correct?